src/orange/BoundingBoxIO.json.hh

```diff
@@ -22,7 +22,7 @@ namespace detail
 //---------------------------------------------------------------------------//
 //! Replace "max" with "inf" since the latter can't be represented in JSON.
 template<class T>
-inline void fix_inf(celeritas::Array<T, 3>* point)
+inline void fix_inf(typename celeritas::BoundingBox<T>::Real3* point)
```

Comment:
Why the change to make this more complicated? I don't get see how using `typename BoundingBox::Real3` improves readability or code safety. For more than a decade C++ has been steadily moving away from explicit types (*especially* when you have to use `typename`).

Reply:
Sorry, this was an accident that happened while trying to resolve merge conflicts